### ISSUE TITLE
region playlist does not retain playlists bug

### ISSUE THREAD

**Issue by jnlundberg** [reaper-oss/sws#1744]
After a few minutes all playlists are gone from the Region Playlist. I love the region playlist, I hope there is a fix. Should I try out the beta?

Reaper v6.78 - March 14 2023
SWS v2.12.1.3 featured build (February 6, 2021)
Mac OSX Monterery 12.6.3
MacBookPro 2015 15"

**Comment by cfillion** [comment]
Could be related to #1743 (region playlist and track notes use a similar storage mechanism), first seen at the beginning of the month. There's also been a very suspicious [crash report](https://forum.cockos.com/showthread.php?p=2659581) (SWS auto-color, same data storage method) pointing to that.

Need to find a reliable way to trigger the data loss to investigate. Playlist didn't disappear when I created 2 region playlists in a blank project with 2 regions, closed the playlist window, waited 15 minutes, re-opened the playlist window.

> Should I try out the beta?

Generally yes: we've made a lot of fixes since 2.12. And if the bug ever causes a crash it would be easier to look into reports made with the latest version rather than a 2 years old version.
